### PR TITLE
dev: clean up dependencies

### PR DESCRIPTION
**Pull request by lambda-0x**
fix: #1929

### CODE REVIEW

File: crates/katana/core/Cargo.toml [hunk 1]
```diff
@@ -62,7 +61,6 @@ messaging = [
     "alloy-provider",
     "alloy-rpc-types",
     "alloy-signer",
-    "alloy-signer-wallet",
```
Comment (lambda-0x) [inline crates/katana/core/Cargo.toml:65]: @kariy can you confirm that this is not needed.

Comment (glihm) [reply crates/katana/core/Cargo.toml:65]: Alloy crates are a bit modified in #1870, but may adjusted during a rebase.

**Review by glihm — COMMENTED** [review]
@lambda-0x if you could rebase that, let's merge it. No need to port this to the namespace branch after some checks. 👍

**Review by glihm — APPROVED** [review]
thanks for this great cleanup mate.

**Pull request merged**